### PR TITLE
remove keys froms tags after delete

### PR DESCRIPTION
**Pull request by betd-sthibault**

### CODE REVIEW

File: Client/RedisClient.php [hunk 1]
```diff
@@ -95,9 +94,18 @@ public function removeKeysFromTags(array $tags)
             $mergedKeys = is_array($currentKeys) ? array_merge($mergedKeys, $currentKeys) : $mergedKeys;
 
         }
+
+
```
Comment (pdenis) [inline Client/RedisClient.php]: empty line

File: Client/RedisClient.php [hunk 2]
```diff
@@ -95,9 +94,18 @@ public function removeKeysFromTags(array $tags)
...
         // execute key removal on unique keys and return them
         $mergedKeys = array_unique($mergedKeys);
+
         $nbDeleted = $this->executeEval('del', $mergedKeys);
+        // remove keys from tags
+        foreach($tags as $intersectExpression) {
+            foreach ($intersectExpression as $tag) {
+                $this->executeEval('SREM',array_merge(array($tag), $mergedKeys));
```
Comment (pdenis) [inline Client/RedisClient.php]: a space is needed after ","

**Comment by betd-sthibault** [comment]
@pdenis j'ai passé un coup de php-cs-fixer ;)

**Comment by pdenis** [comment]
:+1:

**Pull request merged**